### DISCUSSION TITLE
Provide WASM support for Physics

### DISCUSSION THREAD

**Discussion by L1lith** [melonjs/melonJS#1098] [Ideas]
Ideally keep the current system as fallback, and provide WASM integration where available. There's a number of existing libraries for this like Rapier, Ammo.js, etc. This might be easier if the Physics was some kind of modular plugin.

**Comment by obiot** [comment]
actually I was thinking that an easy one to start with this thing is to provide a WASM based implementation for vector and matrix classes, this should (?) be a low hanging fruit and will benefit the physic implementation right away !

**Comment by obiot** [comment]
@L1lith moving this one to discussion since it's more of a larger discussion.
